samples/sample_hevc_fei/src/sample_hevc_fei.cpp

```diff
@@ -701,6 +717,16 @@ void AdjustOptions(sInputParams& params)
         msdk_printf(MSDK_STRING("WARNING: Encoded order is enabled by force in repackctrl.\n"));
         params.bEncodedOrder = true;
     }
+
+    if (!params.bExtBRC && params.TargetKbps) {
+        msdk_printf(MSDK_STRING("WARNING: Target bitrate is ignored as external BRC is disabled\n"));
+        params.TargetKbps = 0;
```

Comment:
TargetKbps is set to 0, so it's not possible to have RateControl =CQP and non-zero TargetKbps at the same time.